helpers/appHelper.js

```diff
@@ -0,0 +1,14 @@
+function findWinner(pokemons, berries) {
+    console.log('Pokemons info before feeding:');
+    pokemons.forEach(element => element.introduce());
+    pokemons.forEach(element => element.feed(berries));
+    console.log('Pokemons info after feeding:');
+    pokemons.forEach(element => element.introduce());
+    pokemons.sort( (a, b) => ((a.experience + a.weight)/a.speed) - ((b.experience + b.weight)/b.speed) );
```

Comment:
`a.experience + a.weight)/a.speed` this small piece of code can be moved to a separate pokemon class method